src/constraints.jl

```diff
@@ -208,3 +208,43 @@ function add_constraint(m::Model, c::AbstractConstraint, name::String="")
     end
     return cref
 end
+
+"""
+    set_rhs(constraint, value)
+
+Set the right-hand side of a constraint. Supported constraints are
+ - ScalarAffineFunction -in- LessThan
+ - ScalarAffineFunction -in- GreaterThan
+ - ScalarAffineFunction -in- EqualTo.
+
+Interval sets are *not* supported.
+
+Note that prior to this step, JuMP will move all variable terms onto the
+left-hand side, and all constant terms onto the right-hand side. For example,
+given a constraint `2x + 1 <= 2`, `JuMP.set_constant(c, 3)` will create the
```

Comment:
Is is called `set_constant` or `set_rhs` ?

Reply:
Oops, I changed back and forth a few times and missed this one.